zetaclient/chains/bitcoin/fee.go

```diff
@@ -35,7 +37,11 @@ const (
 	OutboundBytesMax = uint64(1543) // 1543v == EstimateSegWitTxSize(21, 2, toP2TR)
 	OutboundBytesAvg = uint64(245)  // 245vB is a suggested gas limit for zetacore
 
-	DynamicDepositorFeeHeight = 834500 // DynamicDepositorFeeHeight contains the starting height (Bitcoin mainnet) from which dynamic depositor fee will take effect
+	DynamicDepositorFeeHeight   = 834500 // the mainnet height from which dynamic depositor fee V1 is applied
+	DynamicDepositorFeeHeightV2 = 863400 // the mainnet height from which dynamic depositor fee V2 is applied
+
+	feeRateCountBackBlocks = 2  // the default number of blocks to look back for fee rate estimation
+	defaultTestnetFeeRate  = 10 // the default fee rate for testnet, 10 sat/vB
```

Comment:
Can we follow go idiomatic for comments?
```
// DynamicDepositorFeeHeight is the mainnet height from which dynamic depositor fee V1 is applied
DynamicDepositorFeeHeight = 834500 
```
Also unclear, why do we need this height? What do we only have mainnet and not testnet?

Reply:
zetaclients might vote on different ballot identifiers if deposits happen during the upgrading. Different binary would result different amounts after fee deduction. This could leave the inbound pending for long time until some one report to us.